.github/ISSUE_TEMPLATE/request.md

```diff
@@ -1,9 +1,9 @@
 ---
 name: Request
 about: This is a request for work
-title: '[REQUEST]: '
-labels: 'request'
-assignees: 'leohhhn', 'waymobetta'
+title: "[REQUEST]: "
+labels: request
+assignees: leohhhn, waymobetta
 
```

Comment:
Should this newline be here?

Reply:
It doesn't make a difference but we can remove it.